### ISSUE TITLE
Banners : Add Constraints values for department entry

### ISSUE THREAD

**Issue by SlyMaVoix** [MaVoix/MaVoixSupportGenerator#12]
**TODO**
- maybe a list, because it cannot be a numbered entry
- constraint size of this input (max 3 char)

**Note**
- not a number entry because of Corsica (2A,2B) Damn it! :)

**Comment by ColinMaudry** [comment]
I disagree that we should constrain it to a department. Some local groups are only active at the level of a city, and would consequently like to have a city name there.

In general I'd rather not force certain formats on the user and leave it up to them.

Plus these constrains are based on the default background image, but the user can change this image.

**Comment by SlyMaVoix** [comment]
The label of this entry is "departement number"... 
I think we have a bigger problem here. I think issues for #MAVOIX usage of this tool and you think generic usage which is good. But could we manage this issue only for mavoix branch ?
I mean if we have to think in pure generic way, then it's a banner for event, so we need:
- a background image
- some free edit lines with different Fonts (Title, Subtitles...)

and thats all...
But here we already have specific entries for MAVOIX usage no?

**Comment by ColinMaudry** [comment]
I'm dedicated to the creation of a tool for #MAVOIX users, but I think that what you suggest is too specific for #MAVOIX usage.

- not all groups are department-wide, some are city-wide, thus a "department number" field is too restrictive
- users know how to format a date, and they may want a specific date format. If we want to force a date so that it can automatically calculate the number of days before the elections, it's not really a feature for the user, it's for us.
